src/server/handle-api.js

```diff
@@ -15,26 +15,35 @@ export default ({ server, config }) => {
     path: '/api/v1/{query*}',
     handler: (req, reply) => {
 
-      const base = `${config.siteUrl}/wp-json/wp/v2`
+      const base = `${normalizePath(config.siteUrl)}/wp-json/wp/v2`
```

Comment:
i wonder if node's `path.normalize` could do this work here? pass the full string in? not 100% but could be a good way to do this